dist/backdraft.js

```diff
@@ -1214,42 +1258,68 @@ _.extend(Plugin.factory, {
     // @col: The column from the column manager corresponding to the column we're creating
     //   filtering wrappers for.
     _createFilteringWrappers: function(head, col) {
+      var table = this;
+      table.activeMenu = null;
```

Comment:
Yes on naming it activeFilterMenu.  filterActive is a class used to indicate what image is used for the filter icon (when filters are set for a column vs. when there are no filters set for it).  I'll see if I can clarify these class names.  activeFilterMenu seems good, not sure what to use instead of filterActive/filterInactive.